src/seasidecache.cpp

```diff
@@ -1420,6 +1438,7 @@ void SeasideCache::startRequest(bool *idleProcessing)
             m_fetchRequest.setFilter(favoriteFilter());
             m_fetchRequest.setFetchHint(favoriteFetchHint(m_fetchTypes));
             m_fetchRequest.setSorting(m_sortOrder);
+            qDebug() << "Starting favorites query at" << m_timer.elapsed() << "ms";
```

Comment:
oh, yeah, I misremembered :) but the timing clearly didn't include the real work, it was all 0 or 1ms